### PR TITLE
Updated Pinned Posts with Material-UI

### PR DESCRIPTION
**Pull request by theprover97**
I have improved Pinned-Posts, with minor changes, and the Updates section using Material-UI and function components.

### CODE REVIEW

**Comment by theprover97** [comment]
Screenshot from 2020-04-09 19-28-02

File: src/user/pinned-posts/posts/pinPosts.js [hunk 1]
```diff
@@ -0,0 +1,270 @@
+import React from 'react';
+import {List, Card, Button, Paper, ListItem, ListItemAvatar, Avatar, ListItemText, ListItemSecondaryAction, IconButton, CardMedia} from '@material-ui/core';
+import { makeStyles } from '@material-ui/core/styles';
+import MoreHorizIcon from '@material-ui/icons/MoreHoriz';
```
Comment (Rupeshiya) [inline src/user/pinned-posts/posts/pinPosts.js:4]: destructuring

File: src/user/pinned-posts/posts/posts.js [hunk 2]
```diff
@@ -1,58 +1,20 @@
...
-                official
-              </div>
-            </div>
-          </div>
+export default function Posts(props){
+  return(
+    <div className="posts">
+      <h1>Pinned Posts</h1>
+      <div className="categories">
+        <Button variant="contained" className="btn active">All</Button>
```
Comment (Rupeshiya) [inline src/user/pinned-posts/posts/posts.js:10]: shouldn't it be a tab and filter the result according to the categories?

File: src/user/dashboard/updates/updates.scss [hunk 3]
```diff
@@ -1,33 +1,30 @@
...
-  margin: 20px 0;
-  overflow: auto;
+  margin: 30px 0 0 40px;
   h1 {
-    font-size: 26px;
+    font-size: 28px;
     text-align: center;
     font-weight: bold;
   }
   .org-updates {
```
Comment (Rupeshiya) [inline src/user/dashboard/updates/updates.scss:10]: increase the height of org updates in pinned post section so that it occupy whole left sidebar.

**Comment by Rupeshiya** [comment]
@theprover97 Have you removed the **Organisation update** headline from Dashboard section?
@jaskirat2000 Please look at the new Pinned Post UI

Nice work @theprover97 !!

File: src/user/pinned-posts/posts/posts.js (hunk 2 above)
Comment (theprover97) [reply src/user/pinned-posts/posts/posts.js:10]: Are you asking to enable other buttons to list the 'Donuts', 'Events' and 'Projects' separately?

Comment (Rupeshiya) [reply src/user/pinned-posts/posts/posts.js:10]: yes! Exactly!
If you want you can work on that by opening new issue!

Comment (theprover97) [reply src/user/pinned-posts/posts/posts.js:10]: Sure. I will do that. Thanks!

File: src/user/pinned-posts/posts/pinPosts.js (hunk 1 above)
Comment (theprover97) [reply src/user/pinned-posts/posts/pinPosts.js:4]: @Rupeshiya I think destructuring will increase the bundle-size in the import.

Comment (theprover97) [reply src/user/pinned-posts/posts/pinPosts.js:4]: [Destructuring with Material-UI](https://github.com/op-analytics/NYT-Media-Analytics/issues/134)

Comment (Rupeshiya) [reply src/user/pinned-posts/posts/pinPosts.js:4]: yeah, you are correct!
I really forget that by doing that we will be importing whole core ui material
So leave it @theprover97

Comment (Rupeshiya) [reply src/user/pinned-posts/posts/pinPosts.js:4]: @theprover97 Please undo it.

**Comment by Rupeshiya** [comment]
@theprover97 Please resolve the conflicts.

File: src/user/pinned-posts/posts/pinPosts.js (hunk 1 above)
Comment (theprover97) [reply src/user/pinned-posts/posts/pinPosts.js:4]: OK.

**Comment by theprover97** [comment]
@Rupeshiya conflicts resolved.

File: src/user/dashboard/updates/updates.scss (hunk 3 above)
Comment (theprover97) [reply src/user/dashboard/updates/updates.scss:10]: I have done this in commit f276d1a.

**Comment by Rupeshiya** [comment]
@theprover97 Please resolve the conflict and squash the commit.

**Comment by theprover97** [comment]
@Rupeshiya the 'Resolve Conflicts' tab is disabled although there are conflicting files. Due to this strange behavior, I am unable to resolve the conflicts. The changes I made here are also present in the #416 which has been merged into the 'development' branch. So, I am closing this PR.

**Pull request closed without merging**